src/providers/ObservationProvider.js

```diff
@@ -34,19 +31,18 @@ const ObservationProvider = ( { children }: Props ): Node => {
         setObservationList( localObservations );
       }
 
-      // Live queries and objects emit notifications when something has changed that we can listen for.
-      subscriptionRef.current = localObservations;
+      localObservations.addListener( ( ) => {
+        // changes object has properties including insertions, modifications, and deletions
+        // so we can decide when we need obslist to rerender here. otherwise, it will listen for all changes
+        setObservationList( localObservations.map( o => o ) );
```

Comment:
Do you understand why we need `localObservations.map( o => o )` here instead of just `localObservations`, or using the argument from the `addListener` callback? I did that in the testing branch b/c it worked, but I didn't entirely understand why.

Also, do we need to remove this listener?